cypress/docker-runner.sh

```diff
@@ -5,19 +5,20 @@
 . ./cypress/support/shell-helpers.sh
 
 # Download Fixtures
-curl https://s3.amazonaws.com/cfpb-hmda-public/prod/cypress/fixtures.zip > ./cypress/downloads/fixtures.zip \
-	&& unzip -o cypress/downloads/fixtures.zip -d cypress/
+# curl https://s3.amazonaws.com/cfpb-hmda-public/prod/cypress/fixtures.zip > ./cypress/downloads/fixtures.zip \
+#	&& unzip -o cypress/downloads/fixtures.zip -d cypress/
 
 # Download e2e tests
-curl https://s3.amazonaws.com/cfpb-hmda-public/prod/cypress/e2e.zip > ./cypress/downloads/e2e.zip \
-	&& unzip -o cypress/downloads/e2e.zip -d cypress/  
+# curl https://s3.amazonaws.com/cfpb-hmda-public/prod/cypress/e2e.zip > ./cypress/downloads/e2e.zip \
+#	&& unzip -o cypress/downloads/e2e.zip -d cypress/  
 
 # Download Docusaurus test file
-mkdir -p cypress/e2e/docusaurus
-curl https://raw.githubusercontent.com/cfpb/hmda-combined-documentation/main/cypress/e2e/documentation.cy.js > cypress/e2e/docusaurus/documentation.cy.js
+# mkdir -p cypress/e2e/docusaurus
+# curl https://raw.githubusercontent.com/cfpb/hmda-combined-documentation/main/cypress/e2e/documentation.cy.js > cypress/e2e/docusaurus/documentation.cy.js
```

Comment:
Is there a reason these lines were commented out?

These lines do a few different things.

- Download Fixtures (mostly needed for Filing)
- Download e2e test from production (we upload the e2e tests to s3)
- Download Docusaurus e2e test file from the Docusaurus repo

Reply:
Yes they are moved within the Dockerfile